language/use-cases/prod-catalog-enrichment/genai_prod_catalog_enrichment.ipynb

```diff
@@ -0,0 +1,2071 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #471.    def fix_json(error, context, project):
</pre><p>I think a lot of these prompting functions can be condensed into a single function to make this code easier to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>The idea of having them separate is to keep the code flow simple , and if any pre processing needs to be added for a sepecific prompt it can be done in a stand alone function how it's belong done now instead of a bulk of if and else making the code difficult to follow. Also with the current flow its simple for Developer to switch off any flow by just commenting out the call from the runner module.</p>